pallets/loans/src/loan.rs

```diff
@@ -118,7 +118,7 @@ impl<T: Config> Pallet<T> {
             }
 
             total_borrow_value = currency_borrow_amount
-                .checked_mul(borrow_currency_price)
+                .checked_mul(borrow_currency_price.into_inner())
```

Comment:
```suggestion
                borrow_currency_price.checked_mul_int(currency_borrow_amount)
```

Reply:
Please modify all the places where using `into_inner` in calculating.

Reply:
The parts that were left out before have been corrected